Sustainability/OCP-Sustainability-Profile.json

```diff
@@ -0,0 +1,138 @@
+                                "PhysicalAddress": {
+                                    "ReadRequirement": "Recommended",
+                                    "PropertyRequirements": {
+                                        "ISOCountryCode": {
+                                            "ReadRequirement": "Recommended"
+                                        },
+                                        "ISOSubdivisionCode": {
+                                            "ReadRequirement": "Recommended"
+                                        },
+                                        "PostalCode": {
+                                            "ReadRequirement": "Recommended"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                {
+                    "UseCaseTitle": "Product-level",
+                    "Purpose": "Outer-most Chassis must contain product details and EnivronmentMetrics.",
+                    "UseCaseKeyProperty": "ChassisType",
+                    "UseCaseType": "Normal",
+                    "UseCaseKeyValues": [
+                        "RackMount",
+                        "StandAlone",
+                        "Rack"
+                    ],
+                    "UseCaseComparison": "AnyOf",
+                    "PropertyRequirements": {
+                        "ReadRequirement": "Mandatory",
+                        "PropertyRequirements": {
+                            "Manufacturer": {},
+                            "Model": {},
+                            "EnvironmentMetrics": {},
+                            "Location": {
+                                "ReadRequirement": "Mandatory",
+                                "PropertyRequirements": {
+                                    "PhysicalAddress": {
+                                        "ReadRequirement": "Recommended",
+                                        "PropertyRequirements": {
+                                            "ISOCountryCode": {
+                                                "ReadRequirement": "Recommended"
+                                            },
+                                            "ISOSubdivisionCode": {
+                                                "ReadRequirement": "Recommended"
+                                            },
+                                            "PostalCode": {
+                                                "ReadRequirement": "Recommended"
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "Sensor": {
+        "PropertyRequirements": {
```

Comment:
If the sampling rate and accuracy properties are populated, we want to place minimum values there for this profile.